csrankings-t.csv

```diff
@@ -845,7 +845,7 @@ Tsu-Han Chen,Cornell University,https://www.ece.cornell.edu/people/profile.cfm?n
 Tsu-Jae King Liu,Univ. of California - Berkeley,https://www.eecs.berkeley.edu/~tking,lMUmE_UAAAAJ
 Tsuhan Chen,National University of Singapore,http://www.nus.edu.sg/about/management/chen-tsuhan,syCMBRIAAAAJ
 Tsui-Wei Weng,Univ. of California - San Diego,https://lilyweng.github.io,v8GM4xoAAAAJ
-Tsung-Yi Ho,Chinese University of Hong Kong,https://www.cse.cuhk.edu.hk/people/faculty/tsung-yi-ho/,TRDUYkAAAAAJ
+Tsung-Yi Ho,Chinese University of Hong Kong,https://www.cse.cuhk.edu.hk/people/faculty/tsung-yi-ho/,TRDUYkAAAAAJ

```

Comment:
I'm not sure where this is coming from. Maybe there was a new-line change?

Reply:
Seems fine, don't worry about it.